.github/workflows/cf-worker-example-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Run CF Worker Example App
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on:
+      labels: ubuntu-latest-4-core
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - name: Install protoc for protobuf compilation
+        run: sudo apt-get install -y protobuf-compiler
+      - name: Run Yarn
+        run: yarn --immutable
+      - name: Setup .dev.vars file
+        run: echo "DEVCYCLE_SERVER_SDK_KEY=${{ secrets.DEVCYCLE_SERVER_SDK_KEY }}" > examples/js-cloud-server/cloudflare-worker/.dev.vars
+      - name: Run example app in background
+        run: |
+          yarn nx serve example-js-cloud-server-sdk-cf-worker &
+          echo "SERVER_PID=$!" >> $GITHUB_ENV
+        continue-on-error: true
+      - name: Wait for the server to be up
+        run: sleep 10
+      - name: Test server with curl
+        run: |
+          RESPONSE=$(curl -s http://localhost:8787)  # Replace with your server's port
+          # Check the response or do something based on the result.
+          if [[ "$RESPONSE" != *"DevCycle Variables:"* ]]; then
+            echo "Server didn't return the expected 'DevCycle Variables:' response"
+            exit 1
+          fi
```

Comment:
```suggestion
          RESPONSE=$(curl -s http://localhost:8787)
          if [[ "$RESPONSE" != *"DevCycle Variables:"* ]]; then
            echo "Server didn't return the expected 'DevCycle Variables:' response"
            exit 1
          fi
```

Reply:
Also - would you not want to do more clear sanitizing and checks via jq?

Reply:
we can, I'm more interested in if it actually returns anything and doesn't throw a worker error, which this checks for.